src/main/java/net/sf/jabref/exporter/SaveActions.java

```diff
@@ -52,17 +128,18 @@ private void setAvailableFormatters() {
     }
 
     public BibEntry applySaveActions(BibEntry entry) {
-        for (String key : actions.keySet()) {
-            Formatter formatter = actions.get(key);
-
-            String fieldContent = entry.getField(key);
-            String formattedContent = formatter.format(fieldContent);
-            entry.setField(key, formattedContent);
+        if (enabled) {
```

Comment:
I would change the method signature to return nothing (void). Otherwise the signature suggests that the input entry is not changed and the returned BibEntry contains all the changes.